### PR TITLE
feat(push): adds build and push functionality into different subcommands

### LINKED ISSUE #6: [RFE] Add support for publishing pre-existing artifacts
The client currently supports artifacts building/rendering with an optional push.
To added support for pre-built artifacts, it would make more sense to have two sub-commands; one for building and one for publishing the artifacts.

/kind feature

### PR DESCRIPTION
**Pull request by jpower432**
Closes #6

- [X] Unit tests added
- [X] Docs updated 

Signed-off-by: Jennifer Power <barnabei.jennifer@gmail.com>

### CODE REVIEW

File: cli/build.go [hunk 1]
```diff
@@ -13,37 +13,31 @@ import (
...
+	RootDir string
+	Output  string
 }
 
 var clientBuildExamples = templates.Examples(
 	`
-	# Template content in a directory without pushing
-	client build <directory>
+	# Template content in a directory
+	client build mydirectory
```
Comment (sabre1041) [inline cli/build.go:28]: In the output of the command, do we want to mention where the workspace directory is located?

File: README.md [hunk 2]
```diff
@@ -10,16 +10,43 @@ embedded in UOR artifacts.
...
 # The default workspace is "client-workspace" in the current working directory
-client build <directory> --output my-workspace
+client build mydirectory --output my-workspace
 ```
 
-### Template content in a directory and push to a registry location
-`client build <directory> --push --destination localhost:5000/myartifacts:latest`
+### Push workspace to a registry location
+```
+client push mydirectory localhost:5000/myartifacts:latest
```
Comment (sabre1041) [inline README.md:48]:
```suggestion
client push my-workspace localhost:5000/myartifacts:latest
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Looking good. A few changes

File: cli/build.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpower432) [reply cli/build.go]: Added that info the example text

Comment (jpower432) [reply cli/build.go]: Done.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**